### ISSUE TITLE
v5

### ISSUE THREAD

**Issue by dblodgett-usgs** [dblodgett-usgs/hydrogeoenv#12]
@cheginit -- any changes you want to make in prep for a v5 release? I needed to add nco for a project and have built a new "latest" that just got pushed up. I guess we should wait for the pygeoapi changes so we can get the latest nhdplusTools and HyRiver versions in?

**Comment by cheginit** [comment]
Yes, good idea. I released new versions for all the HyRiver packages except for PyNHD since I am waiting for the PyGeoAPI. Once it's released I'll update the version numbers in hydrogeoenv.

**Comment by dblodgett-usgs** [comment]
Alright -- now that we have the new pygeoapi out, I have more or less cut a version of nhdplusTools. Have some documentation to do, but the release is functionally there.

So this is unblocked from my perspective. 

I added miniconda to the build for python environment setup. Do you think we should move that into the python layer or leave it down in custom?

**Comment by cheginit** [comment]
I'll wrap up the work for the new PyNHD release this weekend.

If you're going to go with miniconda, I would suggest to go with [miniforge](https://github.com/conda-forge/miniforge/). I personally use `mambaforge`. It makes things easier and much faster! I think it's better if you move it to the python layer so all packages can be installed with `mamba`. Mixing `pip` and `conda` is not a good idea, unless a package is no available on `conda-forge`.

**Comment by dblodgett-usgs** [comment]
Paging @rmcd-mscb -- I'm agnostic here. 

I do want a good set of python dependencies pre-installed so we can render notebooks without reinstalling an environment and am totally agnostic to the dependency management system used. My only issue is if it overrides system libraries like gdal and netcdf. These need to remain first on the main users path and not be altered by the python package installation process.

**Comment by rmcd-mscb** [comment]
I don't believe that any conda env will override any system library.  Conda can be activated and deactivated.  I think a user can save an env with the --prefix command to the /mnt/efs/ space so it won't  get wiped out.  miniforge sounds like a good option to me and I'll start using mamba myself :)
